airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -296,6 +296,17 @@
                 "description": "Specifies a query that the connector executes on the source database when the connector sends a heartbeat message. Please see the <a href=\"https://docs.airbyte.com/integrations/sources/postgres/postgres-wal-disk-consumption-and-heartbeat-action-query\">setup guide</a> for how and when to configure this setting.",
                 "default": "",
                 "order": 8
+              },
+              "invalid_cdc_cursor_position_behavior": {
```

Comment:
sounds like for all cdc related connectors we need to have this field defined - is that true? if we have infra to do that in a common place it would be ideal to put it there to make sure the enum/name will be consistent.

Reply:
That's true. I'll see if there is a better place - but some of the descriptions might change due to differences in terminology (WAL/binlog/oplog)